conda/notices/fetch.py

```diff
@@ -38,7 +39,7 @@ def get_notice_responses(
     """
     executor = ThreadPoolExecutor(max_workers=max_workers)
 
-    with Spinner("Retrieving notices", enabled=not silent):
+    with Spinner("Retrieving notices", enabled=not silent, json=context.json):
```

Comment:
To ensure the "retrieving notices" prefix and the progress bar doesn't show up with `--json`.